asr1k_neutron_l3/plugins/l3/schedulers/simple_asr1k_scheduler.py

```diff
@@ -47,17 +49,29 @@ def _get_candidates(self, plugin, context, sync_router):
 
             orig_candidates = plugin.get_l3_agents(context, active=True)
 
+            enabled_candidates = []
+            disabled_hosts = []
+            for candidate in orig_candidates:
+                if not jsonutils.loads(candidate.configurations).get('scheduling_disabled', False):
+                    enabled_candidates.append(candidate)
+                else:
+                    disabled_hosts.append(candidate.host)
```

Comment:
Thinking out loud, as we don't use `orig_candidates` afterwards we could just filter the list. Would keep the diff smaller. On the other hand my short attempt of implementing this didn't really look nicer. Oh well.

Reply:
At first hand I continued naming it `orig_candiates` but felt that the naming was kind of misleading and that `enabled_candidates` would indicate something was done to this list.